Automattic/jetpack
# Connection: Use original flow for Opera Desktop

Fixes #19037

#### Changes proposed in this Pull Request:

* Adds Opera Desktop (modern versions at least) to the Device Detection package.
* Uses the "Safari" check to also check for Opera, as third-parties cookie issues strike there too.

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* In Opera, try to connect a new site before and after the patch.
* Before, you would see the login popup and login, but the original window would never move past the login button.
* After, you're directed to the original flow via Calypso and able to connect.
